### PR TITLE
Add ability to update display name

### PR DESCRIPTION
**Pull request by tsueho**

### CODE REVIEW

File: group.go [hunk 1]
```diff
@@ -204,6 +208,17 @@ func (c *Client) UpdateGroupManagedBy(dn string, managedBy string) error {
 	return c.Conn.Modify(req)
 }
 
+// UpdateGroupDisplayName updates ldap group displayName
+func (c *Client) UpdateGroupDisplayName(dn string, displayName string) error {
+
+	req := ldap.NewModifyRequest(dn, []ldap.Control{})
+
+	req.Replace("displayName", []string{displayName})
```
Comment (pablo-ruth) [inline group.go:216]: Here we need to test if the displayName string is empty and emits a delete request instead of the replace, to correctly handle the case where the displayName attribute is already set and we want to remove it. There is the same logic on the description attribute here: https://github.com/Ouest-France/goldap/blob/4539515cdce09ae136d2a23f32e12ae0556b7f23/group.go#L160-L164

**Review by pablo-ruth — CHANGES_REQUESTED** [review]
Thanks for re-submitting the PR, there is only a little fix on the update func and I can merge it again :)

File: group.go (hunk 1 above)
Comment (tsueho) [reply group.go]: Thanks, done
